docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>I would add a first cell with a main title and a link to open in Colab, e.g. something similar to</p><p> </p><p>(and then also change all other titles one level down to <code>##</code> and <code>###</code> )</p><p> </p><p>It would also be helpful to set the scope of the Colab in the beginning. What can readers expect to learn from this Colab? How does it compare to other Colabs/docs? Does it make most sense to start with this Colab and then move to the readthedocs and then the examples or is there another suggested order?</p><p> </p><pre class="ql-syntax" spellcheck="false"># Flax Basics


&lt;a href="https://colab.research.google.com/github/google/flax/blob/master/docs/notebooks/flax_basics.ipynb" &gt;&lt;img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/&gt;&lt;/a&gt;


Demonstration notebook for Flax.

Code at https://github.com/google/flax
Docs at https://flax.readthedocs.io/
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added the scope part, however I don't think links are useful since those will live in RTD in the end, not as standalone colabs!</p>